apps/apps.humanatlas.io/src/app/pages/us1/cell-population-predictor/cell-population-predictor.component.html

```diff
@@ -0,0 +1,93 @@
+<!-- HEADING -->
+<div class="heading-container">
+  <div>
+    <img [src]="'images/cell-population-predictor-icon.svg' | assetUrl" alt="" />
+  </div>
+  <div class="title-container">
+    <h1>Cell Population Predictor</h1>
+    <hra-software-status-indicator [status]="'Preview'" [size]="'medium'"></hra-software-status-indicator>
+  </div>
+</div>
+
+<!-- INFO -->
+<p class="info-text">
+  Select a 3D tissue extraction site and use the Human Reference Atlas API to predict cell type populations for this
+  site.
+</p>
+
+<!-- CARDS -->
+<div class="cards">
+  <!-- CARD 1 -->
+  <hra-workflow-card title="Upload" class="card card-1">
+    <hra-workflow-card-actions>
+      <mat-icon
+        aria-hidden="false"
+        aria-label="info icon"
```

Comment:
aria-label should be user friendly so update to `Upload data informational tooltip`